src/tests/ftest/harness/config.py

```diff
@@ -17,13 +17,17 @@ class HarnessConfigTest(TestWithServers):
     :avocado: recursive
     """
 
-    def test_harness_config_access_points(self):
-        """Verify the TestWithServers.access_points handling.
+    def test_harness_config(self):
+        """Verify the config handling.
+
+        Verfies the following:
```

Comment:
This was my fault :(
```suggestion
        Verifies the following:
```